tools/cuttlefish-host-image-installer/metapackage-google/usr/share/cf_docker_co_run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Step 1. Build Cuttlefish docker image if there is no such one
+cf_image=$(docker image list | grep "cuttlefish-orchestration")
+if [ "$cf_image" == "" ]; then
+  # Build CF docker image
+  cd /home/vsoc-01
+  if [ ! -d "/home/vsoc-01/android-cuttlefish" ]; then
+    echo "wget https://github.com/google/android-cuttlefish/archive/refs/tags/stable.zip" > android-cuttlefish.log
+    wget https://github.com/google/android-cuttlefish/archive/refs/tags/stable.zip
+    unzip stable.zip
+  fi
+  cd android-cuttlefish-stable/docker/orchestration
+  /bin/bash build.sh &> build.log
+fi
+
+# Step 2. Run CO server
+# use the fixed version instead of HEAD, makes it easier to triage problems when they arise
+CO_VERSION="0.1.0-alpha"
+cd /home/vsoc-01
+if [ ! -d "/home/vsoc-01/cloud-android-orchestration" ]; then
```

Comment:
Ditto, about the directory name `cloud-android-orchestration` and `cloud-android-orchestration-$CO_VERSION`.

Reply:
Thanks! My bug.
Same reason that I used git command before, but it can't work. I am fixing it.